src/lib/replication/repl_dev/raft_repl_dev.h

```diff
@@ -262,14 +275,28 @@ class RaftReplDev : public ReplDev,
     nuraft::ptr< nuraft::log_store > load_log_store() override;
     int32_t server_id() override;
     void system_exit(const int exit_code) override { LOGINFO("System exiting with code [{}]", exit_code); }
+    void become_leader_cb() {
+        auto new_gate = get_last_commit_lsn();
```

Comment:
Shouldn't it be `auto new_gate = raft_server()->get_last_log_index()`? Sorry I didn't get the reason why we are getting the last commit lsn not last log index. Isn't `handle_commit` will be skipped when `lsn > m_rd_sb->durable_commit_lsn` and they will be put in state machine only and waiting for raft server to call commit after it becomes leader?

Reply:
yes you are right